### PR TITLE
updated eda.py and created render_table function

### PR DESCRIPTION
**Pull request by zmerpez**
- Updated eda.py
     * generates 2 tables png and 3 altair charts as html
- Updated README
    * added few links and details, still needs better sentences and exponations
- Added table renderer as a separate .py file, you can use by importing (I can add detail about it to README as well)

### CODE REVIEW

File: src/eda.py [hunk 1]
```diff
@@ -15,27 +15,11 @@
 import os
```
Comment (ssyayayy) [inline src/eda.py:15]: Are we using this package (i.e. os) anywhere? If no, can we remove it?

**Review by ssyayayy — COMMENTED** [review]
Added my comments within the .md file and .py file.

File: src/eda.py (hunk 1 above)
Comment (zmerpez) [reply src/eda.py:15]: yes, only one line: while checking if the path is valid or not.

File: src/eda.py [hunk 2]
```diff
@@ -167,36 +103,125 @@ def generate_feature_overview(data_frame, output_folder, file_name):
...
-    print("generating")
-    generate_dataset_overview(data_frame, output_folder_path, "Table_01")
-    generate_feature_overview(data_frame, output_folder_path, "Table_02")
-    print("done")
+    generate_dataset_overview(data_frame, output_folder_path, "Dataset Overview")
+    generate_feature_overview(data_frame, output_folder_path, "Feature Overview")
+    generate_align_vs_features(data_frame, output_folder_path, "Alignment vs Features")
+    generate_align_vs_year(data_frame, output_folder_path, "Alignment over Time")
+    generate_align_vs_appearances(data_frame, output_folder_path, "Character Appearances by Alignment")
+    print("Succesfull!")
```
Comment (cmmclaug) [inline src/eda.py:219]: Mild typo, not going to hold up merge for this though.

**Review by cmmclaug — APPROVED** [review]
Awesome, great job getting everything done so quickly. Looks solid.

**Pull request merged**